pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Nyman2008/electrolyte_conductivity_Nyman2008.py

```diff
@@ -27,7 +27,7 @@ def electrolyte_conductivity_Nyman2008(c_e, T):
         0.1297 * (c_e / 1000) ** 3 - 2.51 * (c_e / 1000) ** 1.5 + 3.329 * (c_e / 1000)
     )
 
-    E_k_e = 34700
```

Comment:
was this a mistake in the original implementation? 
you could argue we should just not have the arrhenius term in this case

Reply:
Not sure if a mistake, or we decided to keep the Arrhenius term from the Marquis2019 set (originally it all was in `parameters.csv`. However, if now it is specified as a function I think it would be good to stick to what the paper provides, and it this case, Nyman et al. 2008, does not provide temperature dependence so I removed it.

If that makes sense, I will remove the whole Arrhenius term so it is clearer.

Reply:
yeah, that makes sense, just remove the arrhenius term and put in a comment